NetTopologySuite/IO/GML2/GMLWriter.cs

```diff
@@ -84,7 +71,7 @@ public void Write(IGeometry geometry, Stream stream)
         /// </summary>
         /// <param name="coordinate"></param>
         /// <param name="writer"></param>        
-        protected void Write(Coordinate coordinate, XmlTextWriter writer)
+        protected void Write(Coordinate coordinate, XmlWriter writer)

```

Comment:
Will this change work if there are any subclasses compiled against older versions of the library that call the older version of the method?  If not, then this is technically a breaking change, but is that a big deal?